marquez_airflow/utils.py

```diff
@@ -140,3 +143,17 @@ def add_airflow_info_to(task, steps_metadata):
             MARQUEZ_AIRFLOW_VERSION
 
     return steps_metadata
+
+
+def handle_fail(cause):
+    log.debug(f"cause: {cause}")
+
+    return str(cause)
+
+
+def file_to_str(file_name):
+    return open(file=file_name, mode="r").read()
+
+
+def str_to_obj(str):
+    return jsonpickle.decode(string=str)
```

Comment:
if those are used only in tests they should be in the test folder.

Reply:
done.